ts/Core/Chart/StockChart.ts

```diff
@@ -240,8 +238,16 @@ class StockChart extends Chart {
         // Avoid doing these twice
         userOptions.xAxis = userOptions.yAxis = void 0;
 
+        const langOptions = {
+            stockOpen: 'Open',
+            stockHigh: 'High',
+            stockLow: 'Low',
+            stockClose: 'Close'
+        };
+
```

Comment:
1. I think these language strings should be defined in the HLCSeries file rather than StockChart. (O)HLCSeries and candlestick are standalone series types that can also be applied to a vanilla Chart, so technically the lang options are more tied to the series type than the constructor.
2. The second problem is that is actually seems like it doesn't work: https://jsfiddle.net/highcharts/c7zaLto4/ . The tooltip still shows the English terms.